README.md

```diff
@@ -1,56 +1,107 @@
+# Azure Container Compute Upstream
 
-# Contributing
+## Welcome
+As the software development term ["Upstream"](https://en.wikipedia.org/wiki/Upstream_(software_development)) implies, the Azure Container Compute Upstream team is the team at Microsoft responsible for creating and leading the open source initiatives and projects related to container technologies and the [Kubernetes](https://kubernetes.io/) ecosystem. Our team is committed to working with the community, of both contributors and customers, to enhance the overall usabilty experience by helping to build and contribute to useful and valuable open source projects.
 
-This project welcomes contributions and suggestions. If you wonder what the Azure Container Compute team is working on, check out the [Project Board](https://github.com/Azure/container-compute-upstream/projects/1). If you would like to see the Azure Container Compute Upstream team work on something, first check if there is an [existing issue](https://github.com/Azure/container-compute-upstream/issues). If there isn't, please create a new one. If there is existing information elsewhere on the internet, please include the links.
+## Contents
 
-This project is not setup for contributing code. It is intended only for tracking issues.
+* [Upstream Projects Overview](#upstream-projects-overview)
+* [Upstream Project Announcements](#upstream-project-announcments)
```

Comment:
As much as I appreciate this section, I think it will be fairly hard to keep it up to date and very quickly becoming stale. We do have other ways/venues for announcements. Hence my preference would be to remove it. WDYT?